### ISSUE TITLE
Wikipedia doesn’t know what you are

### ISSUE THREAD

**Issue by quinncomendant** [lhorie/mithril.js#829]
You don’t exist on wikipedia yet. This is a feature request.
https://en.wikipedia.org/wiki/Mithril_(JavaScript)

**Comment by pygy** [comment]
​Sadly, I don't think Mithril meets Wikipedia's notability requirements.

The subject of an article must be covered (npt  just mentioned) by
so-called "third party reliable sources" (i.e. books or the mainstream,
technical or academic press; blogs don't count).​

​The criteria for a programming library may be different, I don't know if
there are exceptions in this domain.​

**Comment by quinncomendant** [comment]
That’s very humble of you, but *they* all have wikipedia articles:  ;P

https://en.wikipedia.org/wiki/Ample_SDK
https://en.wikipedia.org/wiki/AngularJS
https://en.wikipedia.org/wiki/Backbone.js
https://en.wikipedia.org/wiki/Chaplin.js
https://en.wikipedia.org/wiki/Dojo_Toolkit
https://en.wikipedia.org/wiki/Ember.js
https://en.wikipedia.org/wiki/Ext_JS
https://en.wikipedia.org/wiki/Google_Closure_Tools
https://en.wikipedia.org/wiki/Handlebars_(template_system)
https://en.wikipedia.org/wiki/JQuery
https://en.wikipedia.org/wiki/KnockoutJS
https://en.wikipedia.org/wiki/Meteor_(web_framework)
https://en.wikipedia.org/wiki/MooTools
https://en.wikipedia.org/wiki/Mustache_(template_system)
https://en.wikipedia.org/wiki/Node.js
https://en.wikipedia.org/wiki/Prototype_JavaScript_Framework
https://en.wikipedia.org/wiki/ReactJS
https://en.wikipedia.org/wiki/Rico_(Ajax)
https://en.wikipedia.org/wiki/Script.aculo.us
https://en.wikipedia.org/wiki/Sencha_Touch
https://en.wikipedia.org/wiki/SproutCore
https://en.wikipedia.org/wiki/Underscore.js
https://en.wikipedia.org/wiki/Wakanda_(software)

BTW, list from the [Template:Application_frameworks](https://en.wikipedia.org/wiki/Template:Application_frameworks), which Mithril should be on as well.

**Comment by veggiemonk** [comment]
Good Idea! Why don't you make one? 
Get inspired form the website and from the README.md ...

**Comment by ArthurClemens** [comment]
The listing of other projects says little.

Eight years ago we tried to get our OS project on Wikipedia. This proved to be tremendously problematic.  Only after sufficient coverage outside of the OS and blogging sphere we got enough points to get approved. I found the entire process very frustrating.

**Comment by quinncomendant** [comment]
Huh, I wasn't aware the creation of new articles was so filtered. 

I had seen many people speak well of Mithril on Twitter and I went first to Wikipedia to learn what the project was. I found the (non-existent) Mithril page linked from https://en.wikipedia.org/wiki/Mithril_(disambiguation). It's likely many others do the same. 

It seems Wikipedia could automate the inclusion process by having a "Vote for this page to exist" button on non-existent pages, and if enough people vote the page is automatically allowed to be created. If enough people want the information it shouldn't matter if "third party reliable sources" are talking about it. 

Sorry no solution here, just a rant. =\

**Comment by pygy** [comment]
You can create an article, but it's likely to get nominated for deletion, then deleted because of the lack of notability according to the rules of Wikipedia.

Off the articles you listed, three have a notice regarding the lack of notability of their subject, and another should too (the Ample SDK, which is totally anecdotic, and its article doesn't have a single reference).

Adding content to Wikipedia is a painful experience. Even [keeping content you know is relevant](https://en.wikipedia.org/wiki/Impromptu_(programming_environment)#Sample_performance) [is painful](https://en.wikipedia.org/wiki/Wikipedia:Files_for_deletion/2011_June_6#File:Study_in_keith.ogv).

**Comment by StephanHoyer** [comment]
Last time I looked in Wikipedia for Software projects was: .... never!

How really cares?

**Comment by impinball** [comment]
### TL;DR: Wikipedia is about 2-3 years behind on the JavaScript world. Don't expect much.

---

Wikipedia has for the longest time had problems with keeping up to date with the JavaScript world. Skip missing fads &ndash; it was missing things that actually stuck in the JavaScript world.

About a year ago, CoffeeScript was nearly the only exclusively compile-to-JS language that [had a page](https://en.wikipedia.org/wiki/CoffeeScript). And that itself was not very large, and it had very few examples, although that has changed recently.

[React's page is scarce of information.](https://en.wikipedia.org/wiki/React_%28JavaScript_library%29) There's no mention of Flux in it. The part on React's data flow model consists of two sentences. The page was half its size a few months ago.

[The MEAN stack's page has practically nothing on it.](https://en.wikipedia.org/wiki/MEAN_%28software_bundle%29) I don't need to explain too much, other than the fact we all know how many blog posts, guides, and other things are out there to explain the stack, the technologies behind it, and best practices regarding it. [The LAMP stack's page dwarfs it in sheer information.](https://en.wikipedia.org/wiki/LAMP_%28software_bundle%29)

[Backbone's page is a stub that's mostly links.](https://en.wikipedia.org/wiki/Backbone.js) That library is five years old at this point, used in many high-profile places. jQuery and Backbone has been a very popular combination for a while, as far back as when Backbone was first just a fad.

[AngularJS's](https://en.wikipedia.org/wiki/AngularJS), and [Ember's](https://en.wikipedia.org/wiki/Ember.js), and [jQuery's](https://en.wikipedia.org/wiki/JQuery) pages are novelties in that they actually have significant amounts of content. jQuery is less surprising, since it's 9 years old at this point. Ember is only 3, but you can't go anywhere without at least hearing about it in passing. It still feels a little bit like a fad, given how everyone talks about it.

If this says anything, none of the JavaScript/ECMAScript articles have much on ES6 developments, despite there being a standard. [Not a *single* thing on arrow functions, ES6 classes, or object shorthands in the article *specifically* focused on JavaScript syntax and semantics.](https://en.wikipedia.org/wiki/JavaScript_syntax) (And IMHO they need merged. About 50% of the content is duplicated.)

**Comment by lhorie** [comment]
Ok, so since there's not anything really actionable here, I'm gonna close this.
